src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Rss20FeedFormatter.cs

```diff
@@ -1019,6 +1020,12 @@ private void ReadXml(XmlReader reader, SyndicationFeed result)
                             }
                         }
                     }
+
+                    if (feedItems.Count != 0)
+                    {
+                        result.Items = feedItems;
+                    }
```

Comment:
Same issue as Atom10FeedFormatter, this could end up being null if feedItems.Count == 0